tests/commons/j2/test_j2_functions.py

```
@@ -11,290 +12,250 @@
+    fmt = regexreplace_fmt("^[^:]*:", "", "all-after-semicolon:is-kept")
+    exp = "is-kept"
+    assert fmt == exp, "unexpected regexreplace result"
+
+
+def test_mapbuild_fn() -> None:
+    assert map_build_fmt is not None, "map_build_fmt function not found"
+    assert isinstance(map_build_fmt, Callable), "map_build_fmt function not callable"
+
+
+def test_mapbuild() -> None:
+    map_expects = dict(a=5, b=6, c=7)
```

Ruff (0.8.2)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)